src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+abstract class Figure implements Drawable, GetArea {
```

Comment:
The class name 'GetArea' is a verb and should be renamed to a noun or noun phrase to better represent what it is, such as 'AreaCalculator'.